TUM Campus App/CardViewController.swift

```diff
@@ -75,7 +74,7 @@ extension CardViewController {
         tableView.rowHeight = UITableViewAutomaticDimension
         imageView.clipsToBounds = true
         tableView.tableFooterView = UIView(frame: CGRect.zero)
-        tableView.separatorColor = UIColor.clear
+        tableView.separatorStyle = .none
```

Comment:
This looks familiar. Is this the same change as in the other PR for the spinner fix?